cocos/asset/assets/material.ts

```diff
@@ -474,7 +474,7 @@ export class Material extends Asset {
     protected _uploadProperty (pass: Pass, name: string, val: MaterialPropertyFull | MaterialPropertyFull[]): boolean {
         const handle = pass.getHandle(name);
         if (!handle) { return false; }
-        const type = Pass.getTypeFromHandle(handle);
+        const type = getTypeFromHandle(handle);
```

Comment:
The static method `Pass.getTypeFromHandle` could not be optimized by terser but getTypeFromHandle could.